### PR TITLE
Reword Volunteer Shifts Tab to be Less Vague + UI underflow

### LINKED ISSUE #625: Reword Volunteer Shifts Tab to be Less Vague + UI underflow
We should change the sequence of tabs to `Upcoming Shifts -> Pending Shifts -> Filled Shifts` so it flows better.

UI underflow: 

Suggestion: Make empty table height the same for minimum height of shifts table

### PR DESCRIPTION
**Pull request by briantu**
…on settings manager tables

## Ticket link

Closes #625 

## Implementation description
* Reworded pending shifts tab from `Requests Pending Confirmation` to `Pending Shifts`
* Fixed shifts table heights to be consistent and background colour of volunteer shifts page to remain `background.light`
* Fixed bottomBorder of last row in settings manager table to be none

## Steps to test
1. Go to `http://localhost:3000/volunteer/shifts`
2. Check that table name was reworded
3. Signup for a shift, approve or cancel shift, check that table heights are the same across all shift tables

## What should reviewers focus on?
* Code correctness

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Review by MatoPlus — COMMENTED** [review]
Seems like the UI "underflow" is still reproducible by playing with zoom settings:
image

**Review by MatoPlus — CHANGES_REQUESTED** [review]
overall for these UI overflow issues, instead of `h="100vh"` we want to set it as a `minH="100vh"` but it seems the UI underflow issue is fixed otherwise.

For this PR, let's change the flex min h to 100vh in `VolunteerShiftsPage` as well as other references of `h="100vh"` to `minH="100vh"` - eg: loading component and not found page

File: frontend/src/components/pages/admin/schedule/SchedulePostingPage.tsx [hunk 1]
```diff
@@ -434,7 +434,7 @@ const SchedulePostingPage = (): React.ReactElement => {
   }
 
   return (
-    <Flex flexFlow="column" width="100%" height="100vh">
+    <Flex flexFlow="column" width="100%" minH="100vh">
```
Comment (MatoPlus) [inline frontend/src/components/pages/admin/schedule/SchedulePostingPage.tsx:437]: I don't think this page should be ever scrollable, investigating.

**Review by MatoPlus — APPROVED** [review]
LGTM, thanks for the cleanup!

File: frontend/src/components/pages/admin/schedule/SchedulePostingPage.tsx (hunk 1 above)
Comment (briantu) [reply frontend/src/components/pages/admin/schedule/SchedulePostingPage.tsx:437]: It's scrollable when we the admin clicks the review button is reviewing the shifts before publishing, so I think we should keep it.

**Pull request merged**